packages/core/src/lib/toast/toast-banner.svelte

```diff
@@ -31,11 +31,11 @@ export { extraClasses as cx };
 
 <div
   class={cx(
-    'relative flex h-10 w-max max-w-[480px] items-center border border-medium bg-medium pl-3 pr-1 text-sm text-default shadow-sm',
+    'relative flex h-10 w-max max-w-[480px] shrink-0 items-center border border-medium bg-medium pl-3 pr-1 text-sm text-default shadow-sm',
```

Comment:
Was this `shrink-0` was supposed to be on the `<Icon>` component?

Reply:
Yes, making the change now.